Lib/shutil.py

```diff
@@ -958,6 +958,8 @@ def unpack_archive(filename, extract_dir=None, format=None):
     if extract_dir is None:
         extract_dir = os.getcwd()
 
+    filename = os.fspath(filename)
```

Comment:
I think it works because it's only passed to other functions that already accept PathLikes. However, it is passed to a handler function that is specific to a file format and maybe some handlers don't accept PathLike, so I'll just add `os.fspath` for it anyway.